src/types/annotator.js

```diff
@@ -110,5 +110,10 @@
  * @typedef {Window & Globals} HypothesisWindow
  */
 
+/**
+ * @typedef {import('../annotator/util/emitter').EventBus} EventBus
```

Comment:
I think it would be fine to continue to reference the `EventBus` type from the `util/emitter` module rather than aliasing it here. This module mainly exists to document types that don't have a natural home in a specific annotator module.

Reply:
There was another reason that I added this here, because it allowed me to define `Emitter` which is not possible to do it within `emitter.js` without exporting the `Emitter` class.

No problem, I will delete these.